src/components/buttons/Button.spec.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Button, {BUTTON_TYPE} from './Button';
+import {shallow} from 'enzyme';
+
+test('render', () => {
```

Comment:
It is the same convention as we use for other the specs, it might be rewritten in the future to `describe` and `it`.